src/cli/init.rs

```diff
@@ -69,6 +75,36 @@ default = { solve-group = "default" }
 {{env}} = { features = {{ features }}, solve-group = "default" }
 {%- endfor %}
 
+[tool.pixi.tasks]
+
+"#;
+
+/// The pyproject.toml template
+///
+/// This is used to create a pyproject.toml from scratch
+const NEW_PYROJECT_TEMPLATE: &str = r#"[project]
```

Comment:
Not 100% sure or what to include in that template. build-system, python version, etc.

Let me know what you think.

Reply:
I know uv has a slower code path for it currently, but would hatch be a better option than setuptools?

Reply:
Maybe. Setuptools seems to remain the most used today. https://framapiaf.org/@fcodvpt/111540079686191842

Reply:
Ok let's leave it.